photon-ml/src/integTest/scala/com/linkedin/photon/ml/normalization/NormalizationTest.scala

```diff
@@ -29,12 +29,13 @@ import org.testng.annotations.Test
 import scala.util.Random
 
 /**
- * All feature normalizations are affine transformation so the resulting models without regularization should be the same
- * except for numerical errors.
- *
- * This test checks the validity of feature normalization in the context of training.
- */
-class NormalizationIntegTest extends SparkTestUtils {
+  * All feature normalizations are affine transformation so the resulting models without regularization should be
+  * the same except for numerical errors.
+  *
+  * This test checks the validity of feature normalization in the context of training.
+  */
```

Comment:
The previous way was correct. Databricks style guide uses Java-style doc blocks:

https://github.com/databricks/scala-style-guide#doc